src/signature.cc

```diff
@@ -92,5 +94,17 @@ string GenerateCheckRequestSignature(const CheckRequest& request) {
   return hasher.Digest();
 }
 
+string GenerateAllocateQuotaRequestSignature(
+    const AllocateQuotaRequest& request) {
+  MD5 hasher;
+  const QuotaOperation& operation = request.allocate_operation();
+  hasher.Update(operation.method_name());
+
+  hasher.Update(kDelimiter, kDelimiterLength);
+  hasher.Update(operation.consumer_id());
```

Comment:
I got a question. Both method_name and metric names from metrics_value_set are from the same method_info data structure.
metrics names can be different for the same method_name?